src/components/ColumnHeader/ColumnHeader.tsx

```diff
@@ -18,9 +19,10 @@ interface Props {
   ariaColIndex: number // aria col index for the header
   ariaRowIndex: number // aria row index for the header
   className?: string // optional class name
+  columnConfig?: ColumnConfig
```

Comment:
it will always be defined, is a ColumnDescriptor, and carries key and index. We could replace with:

```
columnDescriptor: ColumnDescriptor
```

and remove `columnIndex` and `columnName`.

Would it be better, or too obscure?